lxd/network/openvswitch/ovn.go

```diff
@@ -1195,6 +1195,35 @@ func (o *OVN) LogicalSwitchPortAdd(switchName OVNSwitch, portName OVNSwitchPort,
 	return nil
 }
 
+// LogicalSwitchPortIPs returns a list of IPs for a switch port.
+func (o *OVN) LogicalSwitchPortIPs(portName OVNSwitchPort) ([]net.IP, error) {
```

Comment:
This function is inefficient as it makes 2 calls to OVN's northbound (nb) database.

One for `lsp-get-addresses` and agan in `o.LogicalSwitchPortDynamicIPs()`.

It would be more efficient and more consistent to use the same approach used in ` LogicalSwitchIPs` by retrieving both fields in a single request, but just filtered by the port name in question so only a single row is returned.

So you need to change the `--find` argument value so it searches the `name` field I think.